docs/API.md

```diff
@@ -1551,6 +1555,44 @@ Example: `GET ~bob/trusty/wordpress-42/meta/id-series`
 }
 ```
 
+#### GET *id*/meta/home-page
+
+The `meta/home-page` path returns the home page project of 
+the entity if any specified.
+
+```go
+type HomePageResponse struct {
+        HomePage string
+}
+```
+
+Example: `GET bundle/mediawiki/meta/home-page`
```

Comment:
Well, it's a way to do it. DELETE makes it clearer as you're removing a resource. Can be added in later PRs. But do document all possible ways of interaction. Not just GET>